.github/workflows/quality-check-fix.yml

```
@@ -0,0 +1,64 @@
+        with: 
+          node-version: 20
+          cache: 'npm'
+      
+      - name: Install dependencies
+        run: npm ci --prefer-offline --no-audit
+      
+      - name: TypeScript type check
+        # エラーがあっても続行（警告として扱う）
+        run: npm run typecheck || echo "::warning::TypeScript errors found"
+        continue-on-error: true
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)